lib/oli/analytics/by_page.ex

```diff
@@ -20,9 +20,11 @@ defmodule Oli.Analytics.ByPage do
   end
 
   defp get_base_query(project_slug, activity_pages, filtered_sections) do
+    Repo.all(activity_pages)
```

Comment:
I believe you forgot to remove this line since it's performing a query that is never used nor stored in a variable.

Reply:
I remove the IO.inspect but I forgot the repo all 😅